code/go/0chain.net/smartcontract/minersc/miner_eventdb.go

```diff
@@ -7,37 +7,63 @@ import (
 	"0chain.net/smartcontract/dbs/event"
 	"0chain.net/smartcontract/stakepool"
 	"0chain.net/smartcontract/stakepool/spenum"
+	"github.com/0chain/common/core/currency"
 	"github.com/0chain/common/core/logging"
 )
 
-func minerTableToMinerNode(edbMiner event.Miner, delegates []event.DelegatePool) MinerNode {
+type SimpleNodeResponse struct {
+	SimpleNode
+	RoundServiceChargeLastUpdated int64 `json:"round_service_charge_last_updated"`
+}
+
+type DelegatePoolResponse struct {
+	stakepool.DelegatePool
+	RoundPoolLastUpdated int64 `json:"round_pool_last_updated"`
+}
+
+type StakePoolResponse struct {
+	Pools    map[string]*DelegatePoolResponse `json:"pools"`
+	Reward   currency.Coin                    `json:"rewards"`
+	Settings stakepool.Settings               `json:"settings"`
+	Minter   cstate.ApprovedMinter            `json:"minter"`
+}
+
+type MinerNodeResponse struct {
+	*SimpleNodeResponse `json:"simple_miner"`
```

Comment:
Use `node` or `simple_node` as json tag.

Reply:
This terminology was to mirror that of [MinerNode](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/minersc/miner_node.go#L17) in miner_node.go.
```go
// MinerNode struct that holds information about the registering miner.
// swagger:model MinerNode
type MinerNode struct {
	*SimpleNode          `json:"simple_miner"`
	*stakepool.StakePool `json:"stake_pool"`
}

```

Reply:
Should we raise an issue to refactor API field names?